Sources/MapboxCoreNavigation/Feedback/NavigationEventsManager.swift

```diff
@@ -152,7 +152,11 @@ open class NavigationEventsManager {
             preconditionFailure("CFBundleShortVersionString must be set in the Info.plist.")
         }
         mobileEventsManager.initialize(withAccessToken: accessToken, userAgentBase: userAgent, hostSDKVersion: String(describing:stringForShortVersion))
-        mobileEventsManager.sendTurnstileEvent()
+        
+        DispatchQueue.global().async { [weak self] in
+            guard let self = self else { return }
```

Comment:
Fixes:
```
/Users/distiller/project/Sources/MapboxMobileEvents/MMELocationManager.m:60: warning run: This method can cause UI unresponsiveness if invoked on the main thread. Instead, consider waiting for the `-locationManagerDidChangeAuthorization:` callback and checking `authorizationStatus` first.
```

Reply:
I think so, but fix on MapboxMobileEvents will be needed. I've added ticket on their side in https://github.com/mapbox/mapbox-events-ios/issues/348. Will all temporary comment as well.